packages/core/build/webpack.config.js

```diff
@@ -37,15 +37,17 @@ module.exports = function (env) {
             minimizer: MINIMIZERS,
             splitChunks: {
                 chunks: 'all',
-                minSize: 30000,
-                maxSize: 240000,
+                minSize: 102400,
```

Comment:
@ashraf-deriv This single change is producing the same result. 
Can you create a PR with just this change and compare the result with the test link in the PR?

Reply:
I think we can remove all the changes except this one.